hack/ci-common.sh

```diff
@@ -39,7 +39,7 @@ export_artifacts() {
       kubectl -n "$namespace" get pod "$node" --show-managed-fields -oyaml >"$node_dir/pod.yaml" || true
 
       # systemd units
-      for unit in cloud-config-downloader kubelet containerd; do
+      for unit in cloud-config-downloader kubelet containerd containerd-configuration-local-setup; do
```

Comment:
Could you please share how this is related to this PR? I do not see the connection.